app/models/gsa18f/procurement.rb

```diff
@@ -79,6 +79,10 @@ def public_identifier
       "##{proposal.id}"
     end
 
+    def purchaser
+      purchase_step ? purchase_step.user : nil
```

Comment:
even though it's longer, I generally prefer the longer version of this for readability

```
if purchase_step
  purchase_step.user
else
  nil
end
```

I do not feel strongly about explicitly returning `nil` since Ruby does that anyway, but I know we've had previous discussions about others liking it.